doozer/doozerlib/cli/get_nightlies.py

```diff
@@ -187,12 +194,31 @@ async def find_rc_nightlies(runtime: Runtime, arches: Set[str], allow_pending: b
 
     async def _find_nightlies(_arch: str):
         # retrieve the list of nightlies from the release-controller
-        rc_url: str = f"{rc_api_url(tag_base, _arch)}/tags"
+        rc_url: str = f"{rc_api_url(tag_base, _arch, private_nightly)}/tags"
         logger.info(f"Reading nightlies from {rc_url}")
 
-        async with aiohttp.ClientSession() as session:
-            async with session.get(rc_url) as resp:
-                data = await resp.json()
+        # Get the token
+        oc_token_result = subprocess.run(["oc", "whoami", "-t"], capture_output=True, text=True)
```

Comment:
we can move this to inside the if block

Reply:
also let's use exectools instead of subprocess.run ?

Reply:
done